src/rest.js

```diff
@@ -38,6 +38,56 @@ export const onResponse = (fn) => {
   instance.interceptors.response.use(undefined, fn);
 };
 
+/**
+ * Fetch the child models of a resource.
+ * @param model The model to find the children of. It must minimally contain _id and _modelType.
+ * @returns A promise that resolves to the list of child models.
+ */
+export const fetchChildren = ({ _id, _modelType }) => {
+  const urlMap = {
+    collection: [`/folder?parentType=collection&parentId=${_id}`],
+    user: [`/folder?parentType=user&parentId=${_id}`],
+    folder: [`/folder?parentType=folder&parentId=${_id}`, `/item?folderId=${_id}`],
```

Comment:
I think we should make this structure a list of Objects instead of just strings so that we can have a structure like:

```
{
url: '/folder',
params: {
   parentType: 'collection',
   parentId: _id,
}
}
```

That way we can add paging params (and whatever else) more easily into the mix

Reply:
Yes that definitely would be a plus.